src/Cdn/Cdn/Models/Endpoint/PSDeliveryRuleUrlPathCondition.cs

```diff
@@ -1,25 +0,0 @@
-﻿// ----------------------------------------------------------------------------------
```

Comment:
@hytao was this type being used anywhere?

Reply:
This type was introduced but never used by customer.

Reply:
This was not an outputType or parameter for any cmdlet. I think it should be fine to remove it.